### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by aashimgarg**
To deinitialize the submodules as the earlier command was not working in the terminal.

### CODE REVIEW

**Comment by AppInventorWorkerBee** [comment]
Can one of the admins verify this patch?

**Comment by ewpatton** [comment]
Also, please note that deinit is only required under certain circumstances, as described in the previous paragraph:

> If you need to switch back to a branch that does contains the Blockly and Closure Library sources in the tree, you will need to run the command:

**Comment by aashimgarg** [comment]
The error i got :-
Use '--all' if you really want to deinitialize submodules

**Comment by ewpatton** [comment]
@aashimgarg The `.` in this case is significant. If you leave it out, you'll get the error message you saw. You should not get the error message if you include the `.` (at least this is what I observe on git 2.25.0--in which case what is your `git --version` if this isn't congruent with your experience?)

**Comment by pavi2410** [comment]
I use this command everytime when switching between branches and it gives no head aches.

`git submodule deinit -f --all`

**Comment by ewpatton** [comment]
I guess that's fair. @aashimgarg Please update this PR to remove the trailing `.` in this case since `--all` will take care of that.

**Comment by aashimgarg** [comment]
What else should i update? @ewpatton

File: README.md [hunk 1]
```diff
@@ -105,7 +105,7 @@ For developers who will be working on Blocky within the context of App Inventor,
 
 If you need to switch back to a branch that does contains the Blockly and Closure Library sources in the tree, you will need to run the command:
 
-    $ git submodule deinit .
+    $ git submodule deinit --all.
```
Comment (pavi2410) [inline README.md]:
```suggestion
    $ git submodule deinit --all
```

Comment (aashimgarg) [reply README.md]: ok!

**Pull request merged**